store/migration/10001__init_schema.sql

```diff
@@ -234,7 +238,9 @@ VALUES
         'Default',
         'DEFAULT',
         'UI',
-        'PUBLIC'
+        'PUBLIC',
+        'DISABLED',
+        ''
```

Comment:
I guess we should provide default like {{DB_NAME}} or comment on the meaning if empty value specified.